### PR TITLE
Syndicate Ships PR

### PR DESCRIPTION
**Pull request by Vasily2013**
## About The Pull Request

Adds syndicate ships for missions into the file base. They won’t be in the right file location as of yet, but this is a work in progress

## Why It's Good For The Game
KMC said so.
## Changelog
:cl:
add: Three syndicate ships to use for missions
fix: Fixes the fact that there were no mission ships.
/:cl:

### CODE REVIEW

**Comment by Kmc2000** [comment]
PicsOrItDidntHappen :tm:

Can you also put turf and area passthroughs on the borders of each map? (Look at old-FTL's salvage maps, that's how we'll be doing ours)

**Comment by Vasily2013** [comment]
> PicsOrItDidntHappen ™️
> 
> Can you also put turf and area passthroughs on the borders of each map? (Look at old-FTL's salvage maps, that's how we'll be doing ours)

Maps now have turf and area template_noops covering anything that could reasonably considered space.

File: code/game/area/areas/ruins/templates.dm [hunk 1]
```diff
@@ -10,3 +10,9 @@
 /area/ruin/powered/shuttle/medium_3
 
 /area/ruin/powered/shuttle/medium_4
+
+/area/ruin/powered/NSV13/prisonship
```
Comment (francinum) [inline code/game/area/areas/ruins/templates.dm]: Type paths must be lowercase
```suggestion
/area/ruin/powered/nsv13/prisonship
```

File: code/game/area/areas/ruins/templates.dm [hunk 2]
```diff
@@ -10,3 +10,9 @@
...
+
+/area/ruin/powered/NSV13/trooptransport
```
Comment (francinum) [inline code/game/area/areas/ruins/templates.dm]: Type paths must be lowercase
```suggestion
/area/ruin/powered/nsv13/trooptransport
```

File: code/game/area/areas/ruins/templates.dm [hunk 3]
```diff
@@ -10,3 +10,9 @@
...
+
+/area/ruin/powered/NSV13/gunship
```
Comment (francinum) [inline code/game/area/areas/ruins/templates.dm]: Type paths must be lowercase
```suggestion
/area/ruin/powered/nsv13/gunship
```

**Comment by Vasily2013** [comment]
image
image
image

**Comment by Vasily2013** [comment]
> Type paths must be lowercase
> These changes will also need to be made in the maps.

fixed

**Pull request merged**